src/App.css

```diff
@@ -92,10 +92,6 @@
 #botonFiltrosMap{
   font-size: 1rem;
   color: #2b3bcd;
-  border: solid;
-  border-color: #2b3bcd;
-  border-width: 1px;
-  border-radius: 25px;
```

Comment:
this commit needs a better description, 
- what are you trying to do ? 
- why are you doing it this way ?